### PR TITLE
Add: WP Acceptance

### LINKED ISSUE #45: WP Acceptance integration
As noted in #32, but not covered during the 1.2.0 release, we'll want to work on [WP Acceptance](https://github.com/10up/wpacceptance/) integration.  Open to discussion on multiple [Snapshots](https://github.com/10up/wpsnapshots/) to consider for our testing.

### PR DESCRIPTION
**Pull request by dinhtungdu**
### Description of the Change
Add e2e test using https://github.com/10up/wpacceptance

### Alternate Designs
#155 
I ended up using WP Acceptance because we have already used it for many of our plugins. Besides, both WP Acceptance and Core E2E script use puppeteer.

### Benefits
Add e2e testing.

### Possible Drawbacks
Tests may run slower, but it won't affect our workflow.

### Verification Process
See Travis CI build passed.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/10up/classifai/blob/develop/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my change.
- [x] All new and existing tests passed.

### Applicable Issues
Closes #45

### CODE REVIEW

**Comment by ryanwelcher** [comment]
@dinhtungdu just a couple of items on this, otherwise great work!

File: wpacceptance.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
[the remark this answers is not in the document]
Comment (jeffpaul) [reply wpacceptance.json]: Yes, worth adding to be more accurate

File: composer.json [hunk 2]
```diff
@@ -10,8 +10,8 @@
   "type": "wordpress-plugin",
   "license": "GPLv2",
   "require": {
-    "php": ">=7.0",
-    "yahnis-elsts/plugin-update-checker": "^4.4"
+    "php": ">=7.2",
```
[the remark this answers is not in the document]
Comment (jeffpaul) [reply composer.json]: I don't think so, unless one of the new features in v1.5 requires that now?

[the remark this answers is not in the document]
Comment (dinhtungdu) [reply composer.json]: No, `7.2` is a requirement for WPAcceptance

[the remark this answers is not in the document]
Comment (ryanwelcher) [reply composer.json]: 👍

**Pull request merged**